src/subcommands/create_ecc_key.rs

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Create an ECC key.
+//!
+//! Currently a lot of the parameters of the key generation are hardcoded because it is not clear
+//! on how it will be possible in the future to generate a key from the command line. This is
+//! currently useful for playing with the tool, and demonstrating the use of Parsec.
+//!
+//! This will generate a 2048 bits RSA key pair for signing.
```

Comment:
Will it, though?

Reply:
Yeah the text and content of all of the new files is wrong, planning to change it in a subsequent PR